### PR TITLE
Just cult weapons 

### PR DESCRIPTION
**Pull request by TaralGit**
Readded the Drozd SMG as a loot table gun.
drozd
Drozd

Also added sprites for cult weapons. They only function as ordinary melee weapons though.
cult
Ritual dagger, eldritch blade and unholy halberd sprites

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml [hunk 1]
```diff
@@ -201,6 +201,56 @@
...
   - type: Appearance
 
+- type: entity
+  name: Drozd
+  parent: BaseWeaponSubMachineGun
+  id: WeaponSubMachineGunDrozd
+  description: A janky, ancient looking fully automatic Heavy SMG. Uses .45 magnum ammo.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/SMGs/drozd.rsi
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml:211]: This just doubles the vector which we used to replace the drozd.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml (hunk 1 above)
Comment (TaralGit) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml:211]: The Drozd is supposed to be more of an easter egg than a weapon that is regularly used, kinda like the Decker revolver (hence I added it to the loot table). If I knew how to I could maybe add some values in the yml to make it more inaccurate or have a lower rate of fire than the Vector, if that's actually possible.

**Review by moonheart08 — CHANGES_REQUESTED** [review]
The cult weapons go unused.

**Comment by TaralGit** [comment]
> The cult weapons go unused.
I know. But it would be still nice to have them at the ready when cult will be implemented. Just like the cult altars, robes, armor, runes and the trap which we're already shipped.

**Comment by Emisse** [comment]
change vector to drozd

**Comment by metalgearsloth** [comment]
> > The cult weapons go unused.
> 
> I know. But it would be still nice to have them at the ready when the cult will be implemented. Just like the cult altars, robes, armor, runes and the trap which we're already shipped.

Whoever makes the cult PR can add it at the time.

**Comment by TaralGit** [comment]
> Replace the vector with the drozd.

Damn. As much as I like the Drozd, the Vector fits a lot more with NT Security. I'll get rid of the Drozd then.

**Pull request closed without merging**